dune/grid/CpGrid.hpp

```diff
@@ -1071,6 +1074,19 @@ namespace Dune
 #endif
         }
 
+        /// \brief The type of the map describing communication interfaces.
+        typedef VariableSizeCommunicator<>::InterfaceMap InterfaceMap;
+
+        /// \brief Get an interface for gathering/scattering data with communication.
+        ///
+        /// The interface can be used with VariableSizeCommunicator and a custom
+        /// index based data handle to scatter (forward direction of the communicator)
+        // and gather data (backward direction of the communicator).
+        const InterfaceMap& cellScatterGatherInterface()
```

Comment:
Maybe it is just me, but I don't really understand the concepts of `InterfaceMap` and `VariableSizeCommunicator<>`. is it possible to explain their purpose in one or two sentences?